ci/test_python.sh

```diff
@@ -10,4 +10,4 @@ flake8 espnet test utils doc;
 autopep8 -r espnet test utils --global-config .pep8 --diff --max-line-length 120 | tee check_autopep8
 test ! -s check_autopep8
 
-LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-}:$(pwd)/chainer_ctc/ext/warp-ctc/build" pytest --cov=espnet
+LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-}:$(pwd)/chainer_ctc/ext/warp-ctc/build" pytest --cov=espnet --durations=0
```

Comment:
How about moving these options to `setup.cfg`  like this
```
[tool:pytest]
addopts = --verbose --cov=espnet --durations=0
```
Maybe putting options in `setup.cfg` and `ci/test_python.sh` is hard to notice.